tool/tsh/tsh.go

```diff
@@ -1042,6 +1043,19 @@ func makeClient(cf *CLIConf, useProfileLogin bool) (*client.TeleportClient, erro
 		}
 		c.AuthMethods = []ssh.AuthMethod{identityAuth}
 
+		// Also create a dummy agent to hold the key. If cluster is in proxy
```

Comment:
If the agent is a dummy agent, why are keys being loaded into it?

Reply:
`dummy` was a bad choice of words.
updated to `in-memory`.